install-guide.md

```diff
@@ -51,20 +53,93 @@ sudo pacman -S --needed libunwind glfw-x11 vulkan-devel sox git cmake alsa-lib n
 go to [Continued Install.](/wiki/installation/#continued-install)
 
 #### Continued Install
-2. Clone the repo.
+
+1. Clone the repo.
+
+    ```sh
+    git clone --recursive https://github.com/RPCSX/rpcsx && cd rpcsx
+    git submodule update --init --recursive
+    ```
+
+2. Compile the emulator.
+
+    ```sh
+    cmake -B build && cmake --build build -j$(nproc)
+    ```
+
+### Method 2: WSL
+
+**Install WSL**
+Run Windows Powershell as Administrator:
 
 ```sh
-git clone --recursive https://github.com/RPCSX/rpcsx && cd rpcsx
-git submodule update --init --recursive
+wsl --install
 ```
 
-3. Compile the emulator.
+Reboot your machine
+
+**Install Ubuntu:**
+Run Windows Powershell
 
 ```sh
-cmake -B build && cmake --build build -j$(nproc)
+wsl --install
```

Comment:
I believe this needs `-t Ubuntu` or something similar to install the distro from command line

Reply:
You're right we shouldn't assume everyone has wsl --install -d <DistroName> set to the same value of Ubuntu by default.
I'll change it to specify Ubuntu.

Reply:
Changing to `wsl --install -d Ubuntu` because d is for distro not default.
Oddly -s is for setting a default distro.
--set-default, -s <Distro>
        Sets the distribution as the default.
--terminate, -t <Distro>
        Terminates the specified distribution.